test/routes-acl.js

```diff
@@ -0,0 +1,25 @@
+/* jshint camelcase: false */
+
+var chai = require('chai');
+chai.should();
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+var app = require('../server/server');
+var request = require('supertest')('https://localhost:3001');
+
+var TOKEN_ENDPOINT = '/oauth/token';
+var CLIENT_ID = '123';
+var CLIENT_SECRET = 'secret';
+
+describe.only('Route based ACLs', function() {
+
+  before(require('./start-server'));
+
+  after(function(done) {
+    app.close(done);
+  });
+
+  it('should ...', function() {
```

Comment:
The request is usually an api call with the `access_token`. The oAuth2 resource server will establish the appId/userId from the access token. Such ids can then be used by the ACL.